sql/20201027-routed.sql

```diff
@@ -0,0 +1,38 @@
+ALTER TABLE threads
+ADD COLUMN routed boolean;
+
+
+--- populate routed threads
```

Comment:
I think I just figured out what `routed` means. Can we rename it? Am I understanding correctly that it means `inactive`? Or is it something else?

Reply:
right :)  yeah, i'll rename